.gitignore

```diff
@@ -1,3 +1,5 @@
+training/training_output/
+training/*.zip
```

Comment:
Use `*-model-building-code.zip` to be consistent with the default values that the starter-kit installer adds to this file